### PR TITLE
Feature/house rules

### PR DESCRIPTION
**Pull request by joshdbarton**
Column 2 of Book 4 complete. Lots of typos getting cleaned up from Book 2

### CODE REVIEW

File: book-4-controllers/chapters/house-rules-data-annotations.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+1. Test the form again to make sure you get appropriate feedback for bad input. 
+
+## Practice
+1. Update the `Chore` class and the `CreateChore` component to enforce a minimum `ChoreFrequencyDays` of 1, and a maximum of 14
+    - bonus challenge: use the [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#offering_suggested_values) element to suggest 1,3,7,10, and 14 as options (note that this will have no effect on the allowed values on the server side)
+1. Use the `EmailAddress` data annotation to validate that a registering user has input a valid email for their registration, and use the `MaxLength` attribute to ensure that the username submitted is less than 50 characters.
+1. You have already seen the `Required` data annotation used in data models to make nullable types `NOT NULL` in the database. You can also use to to make data properties required when submitted to the API. Try adding it to properties on the Chore and Registration models to further improve model validation for these types.
```
Comment (gregkorte) [inline book-4-controllers/chapters/house-rules-data-annotations.md:90]: `You can also use to to make data properties required when submitted to the API` 

Is that supposed to be `this to`

File: book-4-controllers/chapters/house-rules-more-features.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# More Features for House Rules
+
+## Updating a Chore
+Convert the `ChoreDetails` component into a form to update the `Chore`. After submitting the form, the handle should trigger a navigation back to the chores list view. 
+
+## Highlighting Overdue Chores
+Once a chore has not been completed for as many days as its `ChoreFrequencyDays` it is time to do the chore again! Update the application so that overdue chores' names are red in the `ChoresList` so that users will be aware that they need attending to. To complete this feature, you may find it useful to:
+1. Create a calculated property that determines whether the most recent completion of a chore was completed more than the `ChoreFrequencyDays` ago. To do this you may find the `Max` Linq method useful, as well as using `Include` to get `ChoreCompletions` when retrieving chores. The `AddDays` methods will also come in handy, along with `DateTime.Today`
```
Comment (gregkorte) [inline book-4-controllers/chapters/house-rules-more-features.md:8]: The `AddDays` methods to The `AddDays` method ??

**Review by gregkorte — CHANGES_REQUESTED** [review]
There were just a couple of things I saw initially. Everything else looks good.

File: book-4-controllers/chapters/house-rules-data-annotations.md (hunk 1 above)
Comment (joshdbarton) [reply book-4-controllers/chapters/house-rules-data-annotations.md:90]: 👍

File: book-4-controllers/chapters/house-rules-more-features.md (hunk 2 above)
Comment (joshdbarton) [reply book-4-controllers/chapters/house-rules-more-features.md:8]: 👍

**Review by gregkorte — APPROVED** [review]
:shipit:

**Pull request merged**